### ISSUE TITLE
Test accuracy with the pretrained model

### ISSUE THREAD

**Issue by Simon4Yan** [hyk1996/Single-Human-Parsing-LIP#8]
Hi, thanks for your project.

I test your code with the provided densenet model. I get, 

 overall acc. : 0.457065 
 mean acc.    : 0.109814 
 mean IoU     : 0.054742 

I only changed the LIP dataset path, do you have any suggestions about this issue? I might miss something.

Thanks.

The above issue is using batch>1. After changing back to test batch_size=1, the accuracy is normal. After training a ResNet-50 model, the accuracy is,
 overall acc. : 0.824020 
 mean acc.    : 0.600506 
 mean IoU     : 0.518317 

Moreover, by using a larger batch size (64, by default it is 16), the accuracy will be higher.
